dependencies.yaml

```diff
@@ -434,16 +434,25 @@ dependencies:
           # Align nvcomp version with rapids-cmake
           - nvcomp==4.1.0.6
     specific:
-      - output_types: [requirements, pyproject]
+      - output_types: pyproject
```

Comment:
Why remove `requirements`? I think that `requirements` is used by devcontainers -- so adding the matrix entry `use_cuda_wheels: "false"` may not be having the intended effect.

Reply:
`requirements` is exactly what's used by devcontainers, to create a list of things to install when creating an environment (as it then does `pip wheel --no-build-isolation` and `pip install -e --no-deps`).

I was thinking that dropping `requirements` is safe (I don't know of anything else that needs `nvidia-nvcomp` to show up in `requirements` output for cuDF), and offers a little extra protection against it showing up in the environments that devcontainers create.

But happy to put it back if you'd prefer. `use_cuda_wheels` **should** be sufficient to keep `nvidia-nvcomp` out of both package metadata and the environment for devcontainers.

Reply:
I think we agree on the ramifications. I would prefer for requirements.txt to “make sense”, which means that if someone generated it manually, it would have all the dependencies. Just because devcontainers are the only consumer today doesn’t necessarily means it’s the only possible consumer. Since `use_cuda_wheels` is the designated control point specifically for devcontainers, let’s revert this change.

Reply:
Ok, fine with me.